shoop/xtheme/views/forms.py

```diff
@@ -24,17 +25,20 @@ def populate(self):
         """
         Populate the form with fields for size and plugin selection.
         """
-        sizes = ["sm", "md"]  # TODO: Parametrize? Currently Bootstrap dependent.
-        sizes.extend(set(self.layout_cell.sizes) - set(sizes))
-        self.sizes = sizes
-        for size in self.sizes:
-            self.fields["size_%s" % size] = forms.IntegerField(
-                label=size.upper(),
-                required=False,
-                min_value=0,
-                max_value=12,  # TODO: Parametrize? Currently Bootstrap dependent.
-                initial=self.layout_cell.sizes.get(size)
-            )
+        cell_width_choices = [
+            (3,_("25%")),
+            (4,_("33.33%")),
```

Comment:
The iteration of the value went from the original `1/4 width (25%) col-sm-3 (3/12)` to this value (only percentage). @juhakujala will make the final decision and lets me know :)